PointOfDispensingApp/src/brokerRequests.js

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+
+// define functions for API requests here
+export default {
+  // Search patient for patient retrieval
+  searchPatient: (data) => {
+    let query = `?family=${data.lastName}&given=${data.firstName}&birthdate=${data.birthDate}&address-postalcode=${data.postalCode}`;
+    return axios.get(`http://localhost:3000/Patient${query}`);
+  },
+
+  // Get patient from QR code for patient retrival
+  getPatient: (data) => {
```

Comment:
Why is this hard-code data return here?

Reply:
The card focuses on setting up the workflow for connecting the vue app to the broker, not actually making the correct API calls. I had already worked on the search feature before I realized, which is why the search calls the correct endpoints and returns the data.